protocol/streaming/full_node_streaming_manager.go

```diff
@@ -372,6 +407,24 @@ func toSubaccountStreamUpdates(
 	return streamUpdates
 }
 
+func toPriceStreamUpdates(
+	priceUpdates []*pricestypes.StreamPriceUpdate,
+	blockHeight uint32,
+	execMode sdk.ExecMode,
+) []clobtypes.StreamUpdate {
+	streamUpdates := make([]clobtypes.StreamUpdate, 0)
+	for _, update := range priceUpdates {
+		streamUpdates = append(streamUpdates, clobtypes.StreamUpdate{
+			UpdateMessage: &clobtypes.StreamUpdate_PriceUpdate{
+				PriceUpdate: update,
+			},
+			BlockHeight: blockHeight,
+			ExecMode:    uint32(execMode),
```

Comment:
For my understanding we always expect `execMode` = `ExecModeFinalize` here right? If so is it worth adding a sanity check?